Source/Core/Prune/DependencyEvaluator.cs

```diff
@@ -46,7 +46,9 @@ protected void AddOutgoing(Declaration newOutgoing)
 
     protected void AddIncoming(Declaration[] declarations)
     {
-      incomingSets.Add(declarations);
+      if (CommandLineOptions.Clo.Prune == CommandLineOptions.PruneMode.Automatic) {
```

Comment:
Why don't we need !ExcludeDep(newIncoming) here?

Reply:
How you use it here?